app/models/manageiq/providers/vmware/infra_manager/inventory/parser.rb

```diff
@@ -52,7 +52,7 @@ def parse_datacenter(object, props)
     dc_hash = {
       :ems_ref      => object._ref,
       :uid_ems      => object._ref,
-      :type         => "EmsFolder",
+      :type         => "Datacenter",
```

Comment:
This was just a copy&paste error from the `parse_folder` method, this matches the current refresh_parser (https://github.com/ManageIQ/manageiq-providers-vmware/blob/master/app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb#L1267)